tests/test_search.py

```diff
@@ -461,15 +461,16 @@ def test_search_and_parse(self, shortname, string, expected, settings=None):
                     January UTC
                     June 5 am utc
                     June 23th 5 pm EST
-                    May 31, 8am UTC""",
+                    May 31
+                    8am UTC""",
               [('May 2020', datetime.datetime(2020, 5, datetime.datetime.utcnow().day, 0, 0)),
-               ('June 2020', datetime.datetime(2020, 6, datetime.datetime.utcnow().day, 0, 0)),
-               ('2023', datetime.datetime(2023, 6, datetime.datetime.utcnow().day, 0, 0)),
-               ('January UTC', datetime.datetime(2023, 1, datetime.datetime.utcnow().day, 0, 0, tzinfo=pytz.utc)),
-               ('June 5 am utc', datetime.datetime(2023, 6, 5, 0, 0, tzinfo=pytz.utc)),
+               ('June 2020', datetime.datetime(2020, 6, 30, 0, 0)),
```

Comment:
Won’t this cause these tests to fail whenever the tests are run on a day that is not the 30th of the month? If these tests failed for you, I wonder if maybe we should get here the current date (by local timezone) instead of the UTC day.

Reply:
Corrected test's from the `30` to `datetime.datetime.utcnow().day` 

Regarding the UTC specific time zones are only specified if the input specifically states it